nntrainer/compiler/recurrent_realizer.cpp

```diff
@@ -158,7 +157,6 @@ static void propagateTimestep(LayerNode *node, unsigned int time_step,
   auto is_recurrent_type = [](LayerNode *node) {
     return node->getType() == RNNCellLayer::type ||
            node->getType() == LSTMLayer::type ||
-           node->getType() == LSTMCellLayer::type ||
            node->getType() == ZoneoutLSTMCellLayer::type ||
```

Comment:
Zoneout should be removed as well?
If this is working, does zoneout still takes time_steps?

Reply:
Discussed offline that this is required for now as zoneout mask is being shared.

Later, we can remove timestep as zoneout mask will be held individually.

Reply:
As discussed offline, for now zoneout needs timestep for the layer unittest